scripts/zones/Mog_Garden/TextIDs.lua

```diff
@@ -2,16 +2,16 @@
 
 -- General Texts
 ITEM_CANNOT_BE_OBTAINED  = 6379; -- You cannot obtain the item <item> come back again after sorting your inventory
-ITEM_CANNOT_BE_OBTAINEDX = 6379;
+ITEM_CANNOT_BE_OBTAINEDX = 6378; -- (FIXME: Is this right??) You cannot obtain the <item>.
           ITEM_OBTAINED  = 6384; -- Obtained: <item>
         ITEM_OBTAINEDX   = 6390; -- You obtain <number> <item>!
            GIL_OBTAINED  = 6385; -- Obtained <number> gil
        KEYITEM_OBTAINED  = 6387; -- Obtained key item: <keyitem>
     NOT_HAVE_ENOUGH_GIL  = 6389; -- You do not have enough gil
-MOGLOCKER_MESSAGE_OFFSET = 7458;
+MOGLOCKER_MESSAGE_OFFSET = 7508; -- (FIXME: Is this right??) Your particular paid period of Mog Locker patronage has been extended until the following time, kupo!
```

Comment:
Had to take another guess here as well, since it had no comment or clues as to which mog locker line is required.